plenum/server/request_handlers/handler_interfaces/write_request_handler.py

```diff
@@ -52,9 +52,8 @@ def update_state(self, txn, prev_result, is_committed=False):
         """
         pass
 
-    @abstractmethod
     def gen_state_key(self, txn):
```

Comment:
I think we need to keep this method and use it as the main (primary) key in the state.
It needs to be abstract, so that every req handler must implement it.